.github/workflows/ecosystem-submission.yml

```diff
@@ -9,7 +9,7 @@ name: Ecosystem | Submission check
 
 on:
   issues:
-    types: [opened]
+    types: [opened, edited, reopened]
 #  issue_comment:
 #    types: [created]
```

Comment:
When we init the project we though about regen the submission when using a certain comment and so that's this part commented. I don't why it has been commented over time

Reply:
Good spot, I'll remove them :)